src/theme/themes/utils/generate-theme.scss

```diff
@@ -23,6 +23,7 @@
     "color-red": #eb445a,
     "color-green": #92cb44,
     "color-gray-light": $gray,
+    "color-shade": rgba(black, 0.07),
```

Comment:
nit(non-blocking)
I don't think this colour is defined in such a way that it is very clear when/where it's used. It's not really a named color in the same way that others are, and also when we talk about color shades we usually are referring to the shade of a colour (e.g. light blue), instead of what you are trying to describe as 'the color you should use when you want to provide subtle contrast against a white background). 

I'd probably suggest either you change it to `color-shadow` and use the same color in the `default-box-shadow` below, or leave as a hardcoded color in the templates that are using them. 

Alternatively we could provide a monochrome pallete and have the lightest value as something like `color-black-50` which may or may not line up to something similar.